### DISCUSSION TITLE
侧边栏样式配置问题

### DISCUSSION THREAD

**Discussion by haohaizi134** [vuepress-theme-hope/vuepress-theme-hope#3051] [Q&A] [answered]
### Checklist

- [X] I searched existing issues and no one else requests similar feature.
- [X] I think that 25%+ users are positive towards this feature.

### Describe the feature

我在老版本中看到，PC端的侧边栏是紧贴左边并且可以隐藏收起的。请问一下现在是改了组件了吗？还是说可以自定义配置的呢？
<img width="1391" alt="image" src="https://user-images.githubusercontent.com/35988692/233522661-689c261f-f602-40c5-ba45-456485891244.png">
这是我在JavaGuide哥的文档页看到的侧边栏
<img width="1955" alt="image" src="https://user-images.githubusercontent.com/35988692/233522882-4bbbc6be-17fe-4b0c-8887-ee62902e3f9c.png">

**Comment by Mister-Hope** [comment, marked as answer]
Issue panel is not the place where you can ask questions.

https://theme-hope.vuejs.press/zh/guide/interface/style.html

**Reply by haohaizi134** [reply]
I'm sorry🫠
